third_party/istio-latest/istio-kind-ambient.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  profile: ambient
+  values:
+    global:
+      proxy:
+        clusterDomain: cluster.local
+    pilot:
+      autoscaleMin: 1
+      autoscaleMax: 3
+      cpu:
+        targetAverageUtilization: 60
+    gateways:
+      istio-ingressgateway:
+        autoscaleEnabled: false
+        type: NodePort
```

Comment:
MetalLB works so well with kind - I don't think we want to use NodePort anymore

https://kind.sigs.k8s.io/docs/user/loadbalancer/

Reply:
Yes, we don't want to use NodePort. But current default one (`istio-kind-no-mesh.yaml`) also has the NodePort so if you are alright I will change it in another PR.

Reply:
Sure